packages/core/src/components/blind/blind.scss

```diff
@@ -97,7 +95,6 @@
     position: relative;
     align-items: center;
     flex-grow: 1;
-    flex-shrink: 0;
     margin-right: 1rem;
```

Comment:
```suggestion
    margin-right: 0.5rem;
```

Reply:
Updated, But I can not update snapshot tests, @danielleroux  Can you please update it for me?